packages/labs/react/examples/preact/src/index.tsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/**
+ * @fileoverview This file serves to test that React components made with
+ * `@lit-labs/react` can be used in Preact projects without any type errors.
+ */
+
+import {render} from 'preact';
+import {ElementA} from '@lit-internal/test-elements-react/element-a.js';
+import {
+  ElementEvents,
+  EventSubclass,
+  MyDetail,
+  SpecialEvent,
+} from '@lit-internal/test-elements-react/element-events.js';
+import {ElementProps} from '@lit-internal/test-elements-react/element-props.js';
+
+export function App() {
+  return (
+    <>
+      <ElementA foo="foo" onAChanged={() => {}} />
```

Comment:
Could this example also pass children into the default slot and `footer` slot?

Reply:
Added todo item that'll be fixed in #4142